File: PageBook.AndroidNative/app/src/main/java/ru/maxgog/pagebook/models/EventModel.kt
package ru.maxgog.pagebook.models

import androidx.room.Entity
import androidx.room.PrimaryKey
import kotlinx.datetime.LocalDate

@Entity(tableName = "events")
data class EventModel(
    @PrimaryKey(autoGenerate = true) val id: Int = 0,
    val title: String,
    val description: String,
    val date: LocalDate,
    val time: String
) {
    fun toLocalDate(): LocalDate = date
}

File: PageBook.MAUI/Models/EventModel.cs
using SQLite;

namespace PageBook.Models;

public class Event
{
    [PrimaryKey]
    public string Id { get; set; }
    public string Title { get; set; }
    public string Description { get; set; }
    public DateTime CreatedAt { get; set; }
    public DateTime ModifiedAt { get; set; }
    public string Tags { get; set; }
 }